model/model.go

```diff
@@ -308,22 +309,28 @@ func getNameWithDomain(domain string, name string) string {
 
 func (model Model) SortPoliciesByPriority() error {
 	for ptype, assertion := range model["p"] {
-		for index, token := range assertion.Tokens {
-			if token == fmt.Sprintf("%s_priority", ptype) {
-				assertion.priorityIndex = index
-				break
+		hasPriority := false
+		if _, ok := assertion.FieldIndexMap[constant.PriorityIndex]; !ok {
```

Comment:
Why don't use the newly defined `GetFieldIndex` for all array element fetching?

Reply:
done